scripts/download_cams_pm25.py

```
@@ -0,0 +1,267 @@
+                logger.info(f"\nPM2.5 Statistics:")
+                logger.info(f"  Mean: {pm25_data['value'].mean():.1f} µg/m³")
+                logger.info(f"  Median: {pm25_data['value'].median():.1f} µg/m³")
+                logger.info(f"  Max: {pm25_data['value'].max():.1f} µg/m³")
+                logger.info(f"  Min: {pm25_data['value'].min():.1f} µg/m³")
+        
+    except Exception as e:
+        logger.error(f"Error downloading CAMS data: {e}")
+        raise
+
+
+def main():
```

Ruff (0.12.2)

Missing return type annotation for public function `main`

Add return type annotation: `None`

(ANN201)